Interop/ModelCard/src/test/java/NativeModelsTest.java

```diff
@@ -0,0 +1,162 @@
+import org.tribuo.classification.evaluation.LabelEvaluation;
+import org.tribuo.classification.evaluation.LabelEvaluator;
+import org.tribuo.classification.sgd.linear.LogisticRegressionTrainer;
+import org.tribuo.clustering.evaluation.ClusteringEvaluator;
+import org.tribuo.clustering.example.GaussianClusterDataSource;
+import org.tribuo.clustering.kmeans.KMeansTrainer;
+import org.tribuo.common.libsvm.KernelType;
+import org.tribuo.common.libsvm.SVMParameters;
+import org.tribuo.data.csv.CSVLoader;
+import org.tribuo.datasource.LibSVMDataSource;
+import org.tribuo.evaluation.TrainTestSplitter;
+import org.tribuo.math.optimisers.AdaGrad;
+import org.tribuo.math.optimisers.SGD;
+import org.tribuo.multilabel.MultiLabelFactory;
+import org.tribuo.multilabel.evaluation.MultiLabelEvaluator;
+import org.tribuo.multilabel.sgd.objectives.BinaryCrossEntropy;
+import org.tribuo.regression.RegressionFactory;
+import org.tribuo.regression.Regressor;
+import org.tribuo.regression.evaluation.RegressionEvaluator;
+import org.tribuo.regression.sgd.linear.LinearSGDTrainer;
+import org.tribuo.regression.sgd.objectives.SquaredLoss;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+public class NativeModelsTest {
+    @Test
+    public void testClassificationModelCard() throws IOException {
+        var factory = new LabelFactory();
+        var csvLoader = new CSVLoader<>(factory);
+
+        var irisHeaders = new String[]{"sepalLength", "sepalWidth", "petalLength", "petalWidth", "species"};
+        var irisSource = csvLoader.loadDataSource(Paths.get("src/test/input-data/bezdekIris.data"),"species", irisHeaders);
+        var splitter = new TrainTestSplitter<>(irisSource,0.7,1L);
+
+        var trainData = new MutableDataset<>(splitter.getTrain());
+        var evalData = new MutableDataset<>(splitter.getTest());
+        Trainer<Label> trainer = new LogisticRegressionTrainer();
+        Model<Label> irisModel = trainer.train(trainData);
+
+        var evaluator = new LabelEvaluator();
+        LabelEvaluation evaluation = evaluator.evaluate(irisModel, evalData);
+
+        // create Model Card object and write to file
+        ModelCard modelCard = new ModelCard(irisModel, evaluation);
+        modelCard.addMetric("overall-accuracy", evaluation.accuracy());
+        modelCard.addMetric("average-precision", evaluation.macroAveragedPrecision());
+        modelCard.saveToFile("src/test/output-json/classificationModelCard.json");
+
+        // read file and create Model Card object
+        ModelCard modelCardCopy = new ModelCard("src/test/output-json/classificationModelCard.json");
+        assert(modelCard.toString().equals(modelCardCopy.toString()));
+    }
+
+    @Test
+    public void testMultiLabelClassificationModelCard() throws IOException {
+        var factory = new MultiLabelFactory();
+        var trainSource = new LibSVMDataSource<>(Paths.get(".","src/test/input-data/yeast_train.svm"),factory);
```

Comment:
Ditto for yeast.

Reply:
I've now created a my own mock data as suggested.